requirements.txt

```diff
@@ -0,0 +1,12 @@
+ansible==5.9.0
+ansible-core==2.12.6
+cffi==1.15.0
+cryptography==37.0.2
+Jinja2==3.1.2
+lxml==4.9.1
+MarkupSafe==2.1.1
+packaging==21.3
+pycparser==2.21
+pyparsing==3.0.9
+PyYAML==6.0
+resolvelib==0.5.5
```

Comment:
It seems we do not need all of these packages?

Reply:
I dumped it using pip